adhocteam/Head-Start-TTADP
# TTA-HUB138/Test and Fix Issues found with TP Survey

## Description of change

Krys found an issue while testing the original PR for TTAHUB-138/ Touch Point Survey.

## How to test

Login to the sandbox site. Navigate to the AR Landing Page and click the browser button multiple times. The Region selector should NOT lose its CSS formatting.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy

~- [ ] Update JIRA ticket status~
